apis/swagger.yml

```diff
@@ -1362,6 +1394,93 @@ definitions:
         format: "int64"
         description: "The length of the file dfget requests to download in bytes."
 
+  NetworkInfoFetchRequest:
+    type: "object"
+    description: ""
+    properties:
+      urls:
+        type: "array"
+        description: |
+          The urls is to filter the peer node, the url should be match with taskURL in TaskInfo.
+        items:
+          type: "string"
+
+  Extra:
+    type: "object"
+    description: ""
+    properties:
+      site:
+        type: "string"
+      room:
+        type: "string"
+      rack:
+        type: "string"
+      idc:
+        type: "string"
+
+  TaskFetchInfo:
+    type: "object"
+    description: ""
+    properties:
+      task:
+        $ref: "#/definitions/TaskInfo"
+      pieces:
+        type: "array"
+        description: "The pieces which should belong to the peer node"
+        items:
+          $ref: "#/definitions/PieceInfo"
+
+  Node:
+    type: "object"
+    description: ""
+    properties:
+      basic:
+        $ref: "#/definitions/PeerInfo"
+        description: "Basic node info"
+      extra:
+        $ref: "#/definitions/Extra"
+      load:
+        type: "integer"
+        description: "The load of node, which as the schedule weight in peer schedule"
+      tasks:
+        type: "array"
+        items:
+          $ref: "#/definitions/TaskFetchInfo"
+
+  NetworkInfoFetchResponse:
+    type: "object"
+    description: ""
+    properties:
+      nodes:
+        type: "array"
+        items:
+          $ref: "#/definitions/Node"
+
+  HeartBeatRequest:
+    type: "object"
+    description: ""
+    properties:
+      IP:
+        type: "string"
+        description: "IP address which peer client carries"
+        format: "string"
```

Comment:
Please use `ipv4` or `ipv6` for the format here.

Reply:
done.